Cargo.toml

```diff
@@ -3,11 +3,11 @@ name = "peridio-cli"
 description = "Peridio CLI"
 homepage = "https://peridio.com"
 repository = "https://github.com/peridio/peridio-cli"
-version = "0.22.0-rc.1"
+version = "0.22.1"
```

Comment:
think this should be `0.22.0`.

Reply:
I thought that this would be the version after `custom_metadata`. Unless `custom-metadata` becomes `0.22.1` after the rc is over?

Reply:
Ok, I wasn't sure if you were juggling multiple versions or not. I trust your judgement, just eager to get stuff out.